mu-plugins/central-hub/tests/phpunit/unit/config-store/strStartsWith.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ *  Tests for str_starts_with
+ *
+ * @package    spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @since      1.3.0
+ * @author     Robert A. Gadon
+ * @link       https://github.com/rgadon107/cornerstone
+ * @license    GNU General Public License 2.0+
+ */
+
+namespace spiralWebDb\centralHub\Tests\Unit\ConfigStore;
+
+use function KnowTheCode\ConfigStore\str_starts_with;
+use spiralWebDb\Cornerstone\Tests\Unit\Test_Case;
+
+/**
+ * Class Tests_StringStartsWith
+ *
+ * @package spiralWebDb\centralHub\Tests\Unit\ConfigStore
+ * @group   config-store
+ */
+class Tests_StringStartsWith extends Test_Case {
+
+	/**
+	 * Prepares the test environment before each test.
+	 */
+	protected function setUp() {
+		parent::setUp();
+
+		require_once CENTRAL_HUB_ROOT_DIR . '/src/config-store/internals.php';
+	}
+
+	/*
+	 *  Test that a string starts with a character or substring.
+	 */
+	public function test_that_string_starts_with_a_character_or_substring() {
+		$needle   = 'Hello';
+		$haystack = 'Hello_world!';
+		$actual   = str_starts_with( $haystack, $needle, $encoding = 'UTF-8' );
+
+		$this->assertTrue( $actual );
+		$this->assertSame( true, $actual );
```

Comment:
This whole test needs to be consolidated.

```php
$this->assertTrue( str_starts_with( 'Hello', 'Hello world!' );
```

And let's add more tests than just one.  Check for a single character or other substrings:

```php
$this->assertTrue( str_starts_with( 'Hello', 'Hello world!' );
$this->assertTrue( str_starts_with( 'Hello w', 'Hello world!' );
$this->assertTrue( str_starts_with( 'H', 'Hello world!' );
$this->assertTrue( str_starts_with( 'Cor', 'Cornerstone' );
```